Examples/Examples/PopupExampleView.swift

```diff
@@ -63,12 +54,7 @@ struct PopupExampleView: View {
                     ).first
                     popup = identifyResult?.popups.first
                 }
-                .floatingPanel(
-                    attributionBarHeight: attributionBarHeight,
-                    selectedDetent: $floatingPanelDetent,
-                    horizontalAlignment: .leading,
-                    isPresented: $showPopup
-                ) { [popup] in
+                .sheet(isPresented: $showPopup) { [popup] in
                     PopupView(popup: popup!, isPresented: $showPopup)
                 }
```

Comment:
As part of this PR, should we add some documentation to note the changes in expectations where a `PopupView` can be used?

I'm not sure if an entire DocC article would be warranted, but perhaps an added blurb to the existing doc at the top of PopupView.swift?

Reply:
Added similar for the FeatureFormView conversion in https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/1190/commits/3066cc011c488b0dc2c7ade8a905a2da2d98d527.